xml/System.Data/SerializationFormat.xml

```diff
@@ -38,11 +38,12 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- Microsoft ADO.NET 2.0 introduces the ability to serialize a <xref:System.Data.DataSet> as binary content. Earlier versions of the Microsoft .NET Framework do not support binary serialization.  
-  
- The default format is Xml. This is backward compatible and will work with all versions of the .NET Framework.  
-  
+## Remarks
+
+Microsoft ADO.NET 2.0 introduced the ability to serialize a <xref:System.Data.DataSet> as binary content. Earlier versions of .NET Framework did not support binary serialization. The `Binary` field is obsolete starting in .NET 7, because it's not compatible between .NET Framework and .NET Core/.NET 5+.
```

Comment:
I think the obsoletion of `Binary` has less to do with compatibilities between Framework/Core, and is rather part of the general obsoletion of anything relying on BinaryFormatter (for security reasons).

```suggestion
Microsoft ADO.NET 2.0 introduced the ability to serialize a <xref:System.Data.DataSet> as binary content. Earlier versions of .NET Framework did not support binary serialization. The `Binary` field is obsolete starting in .NET 7, because it relies on <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter>, which is inherently insecure.
```